modules/core/docs/api-reference/parse.md

```diff
@@ -90,7 +90,9 @@ Note that additional data types can be converted to `Response` objects and used
 
 Top-level options
 
-| Option           | Type    | Default   | Description                                                                                                              |
-| ---------------- | ------- | --------- | ------------------------------------------------------------------------------------------------------------------------ |
-| `options.log`    | object  | `console` | By default set to a `console` wrapper. Setting log to `null` will turn off logging.                                      |
-| `options.worker` | boolean | `true`    | If the selected loader is equipped with a worker url (and the runtime environment supports it) parse on a worker thread. |
+| Option             | Type    | Default   | Description                                                                                                              |
+| ------------------ | ------- | --------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `options.metadata` | boolean | `false`   | Currently only implemented for `parseInBatches`, adds initial metadata batch                                             |
+| `options.log`      | object  | `console` | By default set to a `console` wrapper. Setting log to `null` will turn off logging.                                      |
+| `options.worker`   | boolean | `true`    | If the selected loader is equipped with a worker url (and the runtime environment supports it) parse on a worker thread. |
+| `options.CDN`      | string  | -         | Load workers and scripts from this source, defaults to `'https://unpkg.com/@loaders.gl'`                                 |
```

Comment:
Will there still be a `workerUrl` parameter for each loader as well? In some, e.g. Terrain, there's a loader option for the worker url

Reply:
Yes.. nothing has changed in the implementation. Just surfacing existing options. I think this works for libs but not for workers, I will mark it as experimental for now.